### PR TITLE
Implement a basic CLI build

### PR DESCRIPTION
**Pull request by cowboyd**
## Motivation

We want a way to be able to visually represent a graphgen factory so that you can see what kinds of data it will generate for a given set of inputs.

## Approach

We can distribute the tool as a CLI compiled for each individual platform using `deno compile`.

This creates a CLI build for each platform and compiles it with only the ability to access the network, and also to allow reading from the filesystem. Binaries are created for each platform, and uploaded as a github release on any tag starting with `graphgen-cli-vx.x.x`

### CODE REVIEW

File: src/graphql/analyze.ts [hunk 1]
```diff
@@ -309,11 +309,11 @@ function sizeOf(field: GQLField): Size {
...
     assert(directive.arguments, "@size must have arguments");
     let meanArg = directive.arguments?.find((arg) => arg.name.value === "mean");
-    let mean = parseInt((meanArg?.value as graphql.IntValueNode).value ?? 5);
+    let mean = parseInt((meanArg?.value as graphql.IntValueNode)?.value ?? 5);
     let minArg = directive.arguments?.find(({ name }) => name.value === "min");
-    let min = parseInt((minArg?.value as graphql.IntValueNode).value ?? 0);
+    let min = parseInt((minArg?.value as graphql.IntValueNode)?.value ?? 0);
     let maxArg = directive.arguments?.find(({ name }) => name.value === "max");
-    let max = parseInt((maxArg?.value as graphql.IntValueNode).value ?? 10);
+    let max = parseInt((maxArg?.value as graphql.IntValueNode)?.value ?? 10);
```
Comment (cowboyd) [inline src/graphql/analyze.ts:316]: This could be potentially submitted as a separate PR.

**Review by dagda1 — APPROVED** [review]
🚢
